src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {

```

Comment:
Please, add `@Override` annotation in front of all methods that you override.
In that way:
```
@Override
public void doWork() {
...
```

Please, do it for all overrided methods in all classes.